zrcs/zrc-7.md

```diff
@@ -0,0 +1,238 @@
+| ZRC | Title                                    | Status | Type     | Author                              | Created (yyyy-mm-dd) | Updated (yyyy-mm-dd) |
+| --- | ---------------------------------------- | ------ | -------- | ----------------------------------- | -------------------- | -------------------- |
+| 7   | Standardisation of non-fungible metadata | Draft  | Standard | Elliott Green <elliott@zilliqa.com> | 2021-10-11           | 2021-11-26           |
+
+## I. What is Metadata?
+
+Metadata is data that includes infomation about other data, it's widely used to present richer infomation to the end user. Non-fungible tokens can take advantage of a standardised way to present metadata for block explorers, wallets, marketplaces, ecosystem partners and clients to consume.
+
+## II. Abstract
+
+ZRC-7 defines an optional standard for non-fungible tokens to present their associated metadata.
+
+## III. Motivation
+
+This specification builds from the using the existing metadata JSON Schema from [ERC721 - Non-Fungible Token Standard](https://eips.ethereum.org/EIPS/eip-721) and utilises the localization JSON schema from [ERC1155 - Multi Token Standard](https://eips.ethereum.org/EIPS/eip-1155). Several blockchains and ecosystems have defined a richer standard that previously build on ERC721 that provide a better user experience. This specification also takes inspiration from [OpenSea Metadata Standards](https://docs.opensea.io/docs/metadata-standards) to define how non-fungible attributes should be presented and [Algorands Standard Asset Parameters Conventions for Fungible and Non-Fungible Tokens](https://github.com/algorandfoundation/ARCs/blob/main/ARCs/arc-0003.md) to define how ecosystem partners should consume the NFT resources by presenting a related mime type. By standardising the base JSON to ERC-1155's metadata format, it should ensure that bridging of these assets to Zilliqa causes the least friction for existing marketplaces.
```

Comment:
Let's have a section for references. e.g V. References

reference: https://eips.ethereum.org/EIPS/eip-1155#references